trio/_core/_run.py

```diff
@@ -858,8 +858,8 @@ def _return_value_looks_like_wrong_library(value):
                     "trio was expecting an async function, but instead it got "
                     "{!r} – are you trying to use a library written for "
                     "asyncio/twisted/tornado or similar? That won't work "
-                    "without some sort of compatibility shim."
-                    .format(async_fn)
+                    "without some sort of compatibility shim.".
+                    format(async_fn)
```

Comment:
I upgraded yapf (#694), so I think you need to merge from master and then re-run yapf (using 0.24.0) before we can merge. Hopefully that will fix these weird formatting issues too...

Reply:
Yeah, you definitely need yapf 0.24.0